graylog2-web-interface/src/components/common/SearchForm.jsx

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import * as React from 'react';
```

Comment:
Don't we only need to do `* as` when it's Flowtyped?

Reply:
It is not required to add `* as` when using flow, but for a weeks I also prefer importing `React` this way, because we can be sure `React` contains all imports and not only `default`.